assets/javascripts/discourse/components/post-voting-comments-menu.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="post-voting-comments-menu">
+  {{#if this.expanded}}
+    <PostVotingCommentsMenuComposer
+      @id={{@id}}
+      @onSave={{this.handleSave}}
+      @onCancel={{this.closeComposer}}
+    />
+  {{else}}
+    <DButton
+      @display="link"
+      class="post-voting-comment-add-link"
+      @action={{if
+        this.currentUser
+        this.expandComposer
+        (route-action "showLogin")
+      }}
+      @label="post_voting.post.post_voting_comment.add"
+    />
+  {{/if}}
+
+  {{#if (gt @moreCommentCount 0)}}
```

Comment:
I would avoid comparing the values like this in the handlebars template and move it to a getter property in the JS file.

It makes the template harder to read.

Reply:
Agreed, done!